workflows/iuf/operations/update-cfs-config/update-managed-cfs-config.yaml

```diff
@@ -73,27 +73,28 @@ spec:
                     MEDIA_DIR="{{=jsonpath(inputs.parameters.global_params, '$.input_params.media_dir')}}"
                     BOOTPREP_FILE_PATH="{{=jsonpath(inputs.parameters.global_params, '$.input_params.bootprep_config_managed')}}"
                     if [[ -z "$BOOTPREP_FILE_PATH" ]]; then
-                      echo "INFO Cannot access bootprep config file (Input parameter '-bc' not provided). Skipping the 'update-managed-cfs-config' operation"
+                      echo "INFO Cannot access bootprep config file (Input parameter '-bc' not provided). Skipping the 'update-managed-cfs-config' operation" 1>&2
                       exit 0
                     else
-                      echo "INFO Using bootprep config file $BOOTPREP_FILE_PATH for updating CFS config on managed nodes"
+                      echo "INFO Using bootprep config file $BOOTPREP_FILE_PATH for updating CFS config on managed nodes" 1>&2
                     fi
 
                     VARS_FILE_PATH=$(mktemp)
                     cat > $VARS_FILE_PATH <<- 'EOF'
                       {{=toJson(jsonpath(inputs.parameters.global_params, '$.site_params.products'))}}
                     EOF
 
-                    result=$(sat bootprep run \
+                    sat bootprep run \
                       --limit configurations \
                       --overwrite-configs \
                       --vars-file "$VARS_FILE_PATH" \
                       --format json \
-                      $MEDIA_DIR/$BOOTPREP_FILE_PATH 2>&1)
-                    if [ $? -ne 0 ]; then
-                       result=$(echo "$result" | sed -e 's/^/DEBUG /')
-                       echo "ERROR Bootprep for managed nodes failed. Please refer to the operations > configuration_management > Accessing_Sat_Bootprep_Files section of the CSM documentation and then re-run the 'update-cfs-config' stage"
-                       echo -e "DEBUG <sat bootprep run --limit configurations --overwrite-configs --vars-file "$VARS_FILE_PATH" --format json $MEDIA_DIR/$BOOTPREP_FILE_PATH> failed with\n\n$result"
+                      $MEDIA_DIR/$BOOTPREP_FILE_PATH
+                    rc=$?
+                    if [ $rc -ne 0 ]; then
+                      echo "ERROR Bootprep for managed nodes failed. Please refer to the operations > configuration_management > Accessing_Sat_Bootprep_Files section of the CSM documentation and then re-run the 'update-cfs-config' stage" 1>&2
+                      echo "DEBUG <sat bootprep run --limit configurations --overwrite-configs --vars-file "$VARS_FILE_PATH" --format json $MEDIA_DIR/$BOOTPREP_FILE_PATH> failed" 1>&2
```

Comment:
sat bootprep is something users might run so it probably makes sense to change this from DEBUG to ERROR